lambda/input.tf

```
@@ -143,4 +143,10 @@ variable "log_group_retention_period" {
   type        = number
   description = "(Optional) Override the retention period for the lambda log group"
   default     = 14
+}
+
+variable "alias_name" {
+  type        = string
+  description = "(Optional, default 'latest') Alias name"
+  default     = "latest"
```

Could you change this to default to empty and then only create the alias resource if requested?  That way we won't get `latest` aliases auto created for all our functions:
```suggestion
  description = "(Optional, default '') Lambda function's alias name"
  default     = ""
```